src/Persistence/ObjectManager.php

```diff
@@ -9,6 +9,8 @@
 
 /**
  * Contract for a Doctrine persistence layer ObjectManager class to implement.
+ *
+ * @method bool isUninitializedEntity(mixed $value)
```

Comment:
should this be `mixed` or `object` ?

Reply:
The object manager manages objects only, so `object $object` would indeed make sense here imho. It would also be consistent with all other methods on this interface.